crates/router/src/connector/globalpay.rs

```diff
@@ -95,6 +98,97 @@ impl ConnectorCommon for Globalpay {
     }
 }
 
+impl PaymentsCompleteAuthorize for Globalpay {}
+
+impl
+    ConnectorIntegration<
+        api::CompleteAuthorize,
+        types::CompleteAuthorizeData,
+        types::PaymentsResponseData,
+    > for Globalpay
+{
+    fn get_headers(
+        &self,
+        req: &types::PaymentsCompleteAuthorizeRouterData,
+        connectors: &settings::Connectors,
+    ) -> CustomResult<Vec<(String, String)>, errors::ConnectorError> {
+        self.build_headers(req, connectors)
+    }
+
+    fn get_content_type(&self) -> &'static str {
+        self.common_get_content_type()
+    }
+
+    fn get_url(
+        &self,
+        req: &types::PaymentsCompleteAuthorizeRouterData,
+        connectors: &settings::Connectors,
+    ) -> CustomResult<String, errors::ConnectorError> {
+        Ok(format!(
+            "{}transactions/{}/confirmation",
+            self.base_url(connectors),
+            req.request
+                .connector_transaction_id
+                .clone()
+                .ok_or(errors::ConnectorError::MissingConnectorTransactionID)?
+        ))
+    }
+
+    fn get_request_body(
+        &self,
+        _req: &types::PaymentsCompleteAuthorizeRouterData,
+    ) -> CustomResult<Option<String>, errors::ConnectorError> {
+        Ok(Some("{}".to_string()))
```

Comment:
is this empty {} required?

Reply:
yes , we need to send a empty request body.